From 322635513f8b65e48d2220a648c2837eb94c2ab2 Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Wed, 6 Apr 2016 16:24:15 -0500
Subject: [PATCH] Enable python3 support

---
 setup.cfg | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/setup.cfg b/setup.cfg
index 8af1d72..3ae3769 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -15,6 +15,8 @@ classifier =
 	Programming Language :: Python
 	Programming Language :: Python :: 2
 	Programming Language :: Python :: 2.7
+	Programming Language :: Python :: 3
+	Programming Language :: Python :: 3.4
 
 [files]
 packages = 
-- 
2.5.5

